src/components/JiraCard/JiraCard.test.tsx

```diff
@@ -15,15 +15,15 @@
  */
 
 import React from 'react';
-import { render, waitFor } from '@testing-library/react';
-import { ApiRegistry, ApiProvider, UrlPatternDiscovery } from '@backstage/core';
-import { rest } from 'msw';
-import { msw } from '@backstage/test-utils';
-import { setupServer } from 'msw/node';
+import {render} from '@testing-library/react';
+import {ApiProvider, ApiRegistry, UrlPatternDiscovery} from '@backstage/core';
+import {rest} from 'msw';
+import {msw} from '@backstage/test-utils';
+import {setupServer} from 'msw/node';
 // eslint-disable-next-line
-import { MemoryRouter } from 'react-router-dom';
-import { JiraAPI, jiraApiRef } from '../../api';
-import { JiraCard } from './JiraCard';
+import {MemoryRouter} from 'react-router-dom';

```

Comment:
There are a lot of whitespace changes. It's not important in this repo I guess but you might want to configure that in your IDE for other repos that have stricter checks. (I just configured that last week so let me know if you need help)

Reply:
:+1:
Needs to be set up separately for each project it seems. Fixed